docs/scos/dev/upgradability-services/upgradability-guidelines/entity-name-is-not-unique.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Entity name is not unique
+description: Guidelines for making entity names unique
+last_updated: Mar 23, 2022
+template: concept-topic-template
+---
+
+Modules have public and private APIs. While public API updates always support backward compatibility, private API updates can break backward compatibility. So, backward compatibility is not guaranteed in the private API. For example, if you use a core method on the project level, and it is updated or removed with an update, it can cause unexpected issues.
```

Comment:
Please note, that PrivateApi can be updated but the behaviour left without changes for minor's

Reply:
What exactly do you mean?

Reply:
I closed a comment.